src/containers/allelePage/AlleleSequenceView.jsx

```diff
@@ -7,7 +7,8 @@ import useDataTableQuery from '../../hooks/useDataTableQuery';
 function findFminFmax(genomeLocation, variants) {
   let fmax = genomeLocation.end;
   let fmin = genomeLocation.start;
-  if (variants && variants.data.results) {
+
+  if (variants && variants.data && variants.data.results) {
```

Comment:
Just FYI, you should be able to us the ? operator for situations like this. So `variants?.data?.results`